__tests__/pages/ProfilePage.test.js

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import { Enzyme } from "../setup.config";
+import ProfilePage from "<pages>/ProfilePage";
+
+const { shallow } = Enzyme;
+
+describe("ProfilePage", () => {
+  it("should contain h1 element", () => {
+    const profilePageShallow = shallow(<ProfilePage />);
+    expect(profilePageShallow.contains(<h1>welcome to Profile</h1>)).toBe(
```

Comment:
Please change this line to `expect(profilePageShallow.contains(<h1>welcome to Profile</h1>)).toBeTruthy()`